bom-internal/pom.xml

```diff
@@ -130,6 +130,21 @@ POM as their parent.
                 <artifactId>wdl-draft2_2.12</artifactId>
                 <version>${cromwell.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.broadinstitute</groupId>
+                <artifactId>cromwell-wdl-transforms-new-base_2.12</artifactId>
```

Comment:
`new-base` 
any idea why this was done rather than a new straight-up version?

Reply:
> `new-base`
> any idea why this was done rather than a new straight-up version?

The module started with [this commit](https://github.com/broadinstitute/cromwell/commit/cbaf799b6315e5823079cb606806426d9e35685c): "Share new base transforms between WDL 1.0 and 1.1" -- looks like because it is shared, although I haven't paid close attention to their different libraries.